indexer/src/main/java/bio/terra/tanagra/indexing/jobexecutor/JobRunner.java

```diff
@@ -0,0 +1,60 @@
+package bio.terra.tanagra.indexing.jobexecutor;
+
+import bio.terra.tanagra.exception.SystemException;
+import bio.terra.tanagra.indexing.IndexingJob;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+public abstract class JobRunner {
+  protected static final long MAX_TIME_PER_JOB_MIN = 60;
+  protected static final long MAX_TIME_PER_JOB_DRY_RUN_MIN = 5;
+
+  public static final String ANSI_RESET = "\u001B[0m";
```

Comment:
I would call these `TERMINAL_ESCAPE_RESET` or something similar to better indicate what they actually are.